### DISCUSSION TITLE
Can not get Cubase 13 to recognise the JS scrip

### DISCUSSION THREAD

**Discussion by Bad-Wolf-472** [bjoluc/cubase-mcu-midiremote#40] [General]
I had it working yesterday, I was trying to get 2 XTouch extenders working with the Xtouch between them, Best I could do was getting 1 extender and the Xtouch active.  All of a sudden Cubase does not see or load the original JS. I have re-downloaded it fresh, its in the same location, as far as I know the only thing I did different was to activate notepad as the main program to open it, which changed its icon. I got tired of doing the "Open With" thing.  Other than that, I have no clue what Cubase will not load.

**Comment by Bad-Wolf-472** [comment]
Got that resolved. Finally.  Even though I had deleted the script off of drive, it was still listed and disabled in Cubase. Duh.  
Next problem is to get 2 extenders working on either side, so ["extender", "main", "extender"]
Correct?

It may be the midi ins and out that I am confused with,  I will post a screen shot soon.

**Reply by Bad-Wolf-472** [reply]
These are my midi ports currently available for extender, main

![20240520_191715.jpg](https://github.com/bjoluc/cubase-mcu-midiremote/assets/170357041/d5413e9e-e893-4034-ba59-b8662f053b66)

![20240520_191724.jpg](https://github.com/bjoluc/cubase-mcu-midiremote/assets/170357041/f0582edc-ada7-4c6e-8b33-5058ed8402e3)

Now I cant get extender working.  Any advice? Thanks

**Reply by bjoluc** [reply]
Hi @Bad-Wolf-472,
I have released v1.10.0 a couple of days ago which supports auto port detection with multiple extenders: It should work if you connect your left extender via USB and the right one via Ethernet naming the port "X-Touch-Ext2" in rtpMIDI.
